CONTRIBUTING.md

```diff
@@ -231,6 +231,24 @@ Writing separate tests might be much better than making a single test that check
 
 Also, we should use driver methods as much as possible instead of testing commands directly via `RunCommand`.
 
+#### Integration tests naming guidelines
+
+1. Test files should be named after the command they test.
+   For example, `find_test.go` for testing the find command.
+   Test files belonging to the same commands group are organized into folders named after this group,
+   such as `aggregation`, `operations`, `diagnostic`, etc.
```

Comment:
When we add a new folder, do we also need to modify [Taskfile](https://github.com/FerretDB/FerretDB/blob/main/Taskfile.yml#L189) so that newly tests in folder are run? :thinking: 

I think this is a good organisation, but might add complications to adding test, and we want adding tests to be easy.

Reply:
Good point! Overall, there won't be a lot of folders and new folders won't be added too often, but I agree the process of adding new tests/folders should remain simple.

Reply:
This may also have impact on recent work about running integration tests in shard https://github.com/FerretDB/FerretDB/issues/2692

Anyway, I think we need to make sure everyone is onboard, if so then we need to make sure we have tooling in place to do it simply. :hugs:

Reply:
That should not be a problem with [`./...`](https://pkg.go.dev/cmd/go#hdr-Package_lists_and_patterns) as long as test names are unique. Envtool should be able to check that